test/swarm_suite.cpp

```diff
@@ -54,12 +54,14 @@ void test_swarm(test_flags_t const flags)
 {
 	using namespace lt;
 
-	std::printf("\n\n ==== TEST SWARM === %s%s%s%s%s ===\n\n\n"
+	std::printf("\n\n ==== TEST SWARM === %s%s%s%s%s%s%s ===\n\n\n"
 		, (flags & test_flags::super_seeding) ? "super-seeding ": ""
 		, (flags & test_flags::strict_super_seeding) ? "strict-super-seeding ": ""
 		, (flags & test_flags::seed_mode) ? "seed-mode ": ""
 		, (flags & test_flags::time_critical) ? "time-critical ": ""
 		, (flags & test_flags::suggest) ? "suggest ": ""
+		, (flags & test_flags::v1_meta) ? "v1-meta ": ""
+		, (flags & test_flags::v2_meta) ? "v2-meta ": ""
```

Comment:
this isn't used anywhere (yet), but it seems reasonable to have these flags for future tests